gen/gmail1/Cargo.toml

```diff
@@ -23,15 +23,14 @@ mime = "^ 0.3.0"
 serde = { version = "^ 1.0", features = ["derive"] }
 serde_json = "^ 1.0"
 itertools = "^ 0.10"
-google-apis-common = { path = "../../google-apis-common", version = "5.0.1" }
-hyper = "^ 0.14"
+google-apis-common = { path = "../../google-apis-common", version = "6" }
```

Comment:
Please remove all changes to this file as it is generated (and says `# DO NOT EDIT!` in its first line).
Also, please refrain from adding features to `hyper` as that is something the application will control, it's not up to the library to select features unless they are needed for it to function, which I believe isn't the case here.

As you see [in the manifest source file](https://github.com/Byron/google-apis-rs/blob/209d70bfede6a4d0a3d75064fbd2f17d395d0095/src/generator/templates/Cargo.toml.mako#L37), the library is already upgraded, thus the code needs to be regenerated and probably republished to make it come through.

Reply:
Aw shoot, yes, I will remove it. This was the lazy hack to make it work locally :)